prune.go

```diff
@@ -100,6 +100,14 @@ func shouldPrune(oldCount, newCount int, pruneOptions PruneOptions) (bool, strin
 	return false, "transactions have not grown significantly"
 }
 
+func getOracle(db *mgo.Database, txns *mgo.Collection, txnsCount int, maxMemoryTxns int) (Oracle, func(), error) {
```

Comment:
so its simply a hint as to whether we think it is small enough to load into memory. Given we had just done txns.Count() about 5 lines before calling this function, I was avoiding a round-trip to the database for information we already had.